spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/HttpWebServiceMessageSenderBuilder.java

```diff
@@ -35,29 +36,28 @@
  */
 public class HttpWebServiceMessageSenderBuilder {
 
-	private Integer connectionTimeout;
+	private Duration connectTimeout;
```

Comment:
Good catch. I've polished the wrong name in https://github.com/spring-projects/spring-boot/commit/93396aef51ac32cdeb1545f2b7ede923fd9f0efb please rebase and push force an update.